test/framework/setup_test.go

```diff
@@ -0,0 +1,14 @@
+package framework
+
+import (
+	"context"
+	"testing"
+)
+
+// setup has been added after finding compilation errors due to this
+// missing function.
+// TODO check why setup has been missing
+func setup(t *testing.T) context.Context {
```

Comment:
I think the function was renamed from setup to Setup at 
https://github.com/SAP/stewardci-core/blob/master/test/framework/client_creator.go#L19

This should be also adjusted in the tests IMHO